src/network/ssl/socket.c

```diff
@@ -279,8 +284,14 @@ match_uri_host_ip(const char *uri_host,
 	 * network byte order.  */
 	switch (ASN1_STRING_length(cert_host_asn1)) {
 	case 4:
+#ifndef win32
```

Comment:
Ah, sorry about that. This has to be definitely WIN32 not win32. If You'd be able to, please correct me. Luckily it's as if not defined so it doesn't affect the other platforms.